client/my-sites/checkout/src/components/wp-order-review-line-items.tsx

```diff
@@ -409,6 +411,8 @@ function LineItemWrapper( {
 				isSummary={ isSummary }
 				createUserAndSiteBeforeTransaction={ createUserAndSiteBeforeTransaction }
 				responseCart={ responseCart }
+				restorableProducts={ restorableProducts }
+				setRestorableProducts={ setRestorableProducts }
```

Comment:
It seems `LineItemWrapper` doesn't need these values, we can move the hook call inside `LineItem` instead of calling it here and passing the return value down.

Reply:
Adding the hook import on `packages/wpcom-checkout/src/checkout-line-items.tsx` triggers this error:

```
'calypso/my-sites/checkout/src/components/restorable-products-context' import is restricted from being used by a pattern.eslint[no-restricted-imports](https://eslint.org/docs/latest/rules/no-restricted-imports)
```

Reply:
You're right. I didn't realize `LineItem` was in a package.